sdk_contrib/async_processor/src/main/java/io/opentelemetry/sdk/contrib/trace/export/DisruptorEventQueue.java

```diff
@@ -129,6 +130,18 @@ void shutdown() {
     }
   }
 
+  void forceFlush() {
+    if (isShutdown) {
+      return;
+    }
+    synchronized (this) {
+      if (isShutdown) {
+        return;
+      }
+      enqueue(null, EventType.ON_FORCE_FLUSH);
```

Comment:
This is not waiting for the processor thread to actually call the flush maybe just document it for the moment and add a TODO to wait for flush to be called. Or simply create an issue.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/906